### ISSUE TITLE
GCC is not found automatically on my Mac

### ISSUE THREAD

**Issue by arichar6** [USNavalResearchLaboratory/turboWAVE#4]
On my Mac, I've installed gcc via homebrew. However, the easy-install GUI can't find it, and gives this error:

![image](https://user-images.githubusercontent.com/758066/140179860-e671fa58-d9dc-46ff-8351-2ee6e0fb6304.png)

I think the problem is the location where brew installs the gcc tools. On my computer, they were installed in `/opt/homebrew/bin`. So, I ran these commands:
```
sudo mkdir /usr/local/opt
sudo mkdir /usr/local/opt/gcc
sudo ln -s /opt/homebrew/bin /usr/local/opt/gcc/bin
```

After that, the "Compile Code" task worked correctly.

I suspect that this problem is related to the search defined here:

https://github.com/USNavalResearchLaboratory/turboWAVE/blob/aff6f77e53cb26068540c5f2fd746c08b5c134c5/tools/twutils/src/twutils/command_line/twinstall.py#L398

**Comment by dfgordon** [comment]
Is this an "Apple Silicon" Mac by any chance?  According to https://docs.brew.sh/Installation the Intel macs use /usr/local while Apple Silicon uses /opt/homebrew.  We will have to generalize the search it appears.

**Comment by arichar6** [comment]
Yes, this is an M1 (Apple Silicon) Mac.

**Comment by dfgordon** [comment]
Thanks good to know.  It may take some time to fully accommodate them.  I personally don't have one yet.

**Comment by dfgordon** [comment]
Fixed in 4.7.0
